src/components/ec-datepicker/ec-datepicker.story.js

```diff
@@ -53,6 +77,8 @@ export const basic = (args, { argTypes }) => ({
 
       <button @click="toggleWeekendAvailability">Toggle weekend availability</button>
       <button @click="changeDisabledDates('2022-02-22')">Disable 2022-02-22 UTC</button>
+      <button @click="model = new Date('2022-02-25')">Select 2022-02-25 date</button>
+      <button @click="dateFormatBasedOnLocale = 'd/m/Y'">Change date format to 'd/m/Y'</button>
```

Comment:
We could just create a dropdown control with multiple choises like `null`, `'d/m/Y'`, `'d.m.Y'` and `'d-m-Y'`.

Reply:
Done, thanks!